server/src/error.rs

```diff
@@ -86,7 +86,7 @@ impl Display for CertificateError {
 
 #[derive(Debug)]
 pub enum EdgeError {
-    AuthorizationDenied,
+    AuthorizationDenied(String),
```

Comment:
I'd recommend making a new error case for this rather than modifying an existing one. You should be able to get exactly the same behavior with fewer changes and it'll have the nice side effect of being more explicit

Reply:
You mean something like this? https://github.com/Unleash/unleash-edge/pull/474/commits/e39da3970e92cb67d8350fae6a9f8eb30b9550f6

Reply:
Ye! That's nice!